src/components/Button/index.tsx

```diff
@@ -342,18 +303,6 @@ function Button({
     );
 }
 
-Button.propTypes = propTypes;
-Button.defaultProps = defaultProps;
 Button.displayName = 'Button';
 
-const ButtonWithRef = React.forwardRef((props, ref) => (
-    <Button
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...props}
-        forwardedRef={ref}
-    />
-));
-
-ButtonWithRef.displayName = 'ButtonWithRef';
-
-export default withNavigationFallback(ButtonWithRef);
+export default withNavigationFallback(React.forwardRef(Button));
```

Comment:
Can we remove `withNavigationFallback` now? And use navigation hook instead @VickyStash

Reply:
@blazejkustra It looks like it should be possible, @kubabutkiewicz is going to give it a try